distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Expand_Objects_Helpers.enso

```diff
@@ -76,7 +76,6 @@ expand_column (table : Table) (column : Text | Integer) (fields : (Vector Text)
 
      table = Table.new [["aaa", [1, 2]], ["bbb", [[30, 31], [40, 41]]]]
      # => Table.new [["aaa", [1, 1, 2, 2]], ["bbb", [30, 31, 40, 41]]]
-@column Widget_Helpers.make_column_name_selector
```

Comment:
Why don't we need this anymore?

Reply:
Internal method that shouldn't have had it.